app/src/components/fields/CbSelectField.tsx

```diff
@@ -89,7 +90,7 @@ const CbSelectField: React.FC<ICbSelectField> = (props) => {
       {data?.map((a) => {
         const item = typeof a === 'string' ? { label: a, value: a } : { label: a.value, value: a.id };
         return (
-          <MenuItem key={item.value} value={item.value}>
+          <MenuItem disabled={disabledValues?.[item.value]} key={item.value} value={item.value}>
```

Comment:
Does this still work if the value happens to be 0?

Reply:
It should be a string value here, but even if it were the numeric zero that is still a valid index.